tests/network/probes.go

```diff
@@ -106,21 +111,13 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			if disableEnableCycle {
 				By("Disabling the guest-agent")
 				Expect(console.LoginToFedora(vmi)).To(Succeed())
-
-				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
-					&expect.BSnd{S: "sudo systemctl stop qemu-guest-agent\n"},
-					&expect.BExp{R: console.PromptExpression},
-				}, 120)).ToNot(HaveOccurred())
+				Expect(stopGuestAgent(vmi)).ToNot(HaveOccurred())
 
 				// Marking the status to not ready can take a little more time
 				tests.WaitForVMIConditionRemovedOrFalse(virtClient, vmi, v1.VirtualMachineInstanceReady, 300)
 
 				By("Enabling the guest-agent again")
-				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
-					&expect.BSnd{S: "sudo systemctl start qemu-guest-agent\n"},
-					&expect.BExp{R: console.PromptExpression},
-				}, 120)).ToNot(HaveOccurred())
-
+				Expect(startGuestAgent(vmi)).ToNot(HaveOccurred())
```

Comment:
I think `To(Succeed())` fits better here.

Reply:
ditto.